Update VPN_README.md

colindavey opened the pull request:
I tried to clarify some of the things that confused me when going through this process, and clarify where the steps differ between the GUI and command-line instructions.

colindavey committed:
Update VPN_README.md

I tried to clarify some of the things that confused me when going through this process, and clarify where the steps differ between the GUI and command-line instructions.

oliverdain commented on docs/VPN_README.md:
```diff
@@ -13,16 +13,62 @@ without help from Continu. To this end we also have a simple [WireGuard](https:/
+
+NOTE: Steps 1 and 2 can take place in parallel. 
+
+Figure out what the next free IP address is by looking at [the server config file](../ansible/templates/wg0.conf). Note that each user has a different number for the `AllowedIPs` line between `100.65.0.` and `/32`. Look at the number for the last user in the list, and increment that number to select the user's IP address. Send the user this IP address so they can create their configuration. 
```
@colindavey here and below please line-wrap this to 120 characters - the READMEs probably get read via `cat` or `less` on the command line more than they do in a browser so having them line wrapped is really handy.

oliverdain commented on docs/VPN_README.md:
```diff
@@ -13,16 +13,62 @@ without help from Continu. To this end we also have a simple [WireGuard](https:/
+
+Finally, create a pull request to check in the modified `wg0.conf` file.
+
+The master server now has IP address 100.64.0.1 on the WireGuard network.
```
The master server has always had that address. Adding a new peer doesn't change the server's IP. Maybe reword like:

```suggestion
The user can now access the master server at 100.64.0.1 on the WireGuard network.
```

oliverdain reviewed (approved):
Very nice @colindavey . Thanks for doing this. Please line wrap it before merging.

colindavey committed:
Update VPN_README.md

Wrapped the text
Made suggested wording change
Added additional test of connection -- loading url in browser

colindavey committed:
Update VPN_README.md

Formatting bullet list
